source/fundamentals/crud/write-read-pref.txt

```diff
@@ -0,0 +1,253 @@
+       successful writes issued with a write concern of ``majority`` and
+       acknowledged prior to the start of the read operation. For more information, see the
+       :rapid:`Read Concern specification </reference/read-concern-linearizable/#mongodb-readconcern-readconcern.-linearizable->`.
+
+   * - ``Local()``
+     - Specifies that the query should return the instance’s most recent
+       data. For more information, see the
+       :rapid:`Read Concern specification </reference/read-concern-local/#mongodb-readconcern-readconcern.-local->`.
+
+   * - ``Majority()``
+     - Specifies that the query should return the instance’s most recent
+       data acknowledged as having been written to a majority of members
+       in the replica set. For more information, see the
+       :rapid:`Read Concern specification </reference/read-concern-majority/#mongodb-readconcern-readconcern.-majority->`.
+
+   * - ``Snapshot()``
+     - Specifies that the query should return a complete copy of the
+       data in a ``mongod`` instance at a specific point in time. Only
+       available for operations within multi-document transactions. For more information, see the
+       :rapid:`Read Concern specification </reference/read-concern-snapshot/#mongodb-readconcern-readconcern.-snapshot->`.
+       
+You can alternatively specify a read preference in your connection string.
+See the :manual:`Server Manual entry on Read Preference Options </reference/connection-string/#read-preference-options>` for more
+information.
+
+Example
+~~~~~~~
+
+The following code shows how you can specify a read concern of
+"majority" and select a ``Collection`` with this configuration:
+
+.. code-block:: go
+   :emphasize-lines: 1-2
+
+   rc := readconcern.Majority()
+   opts := options.Collection().SetReadConcern(rc)
+   database := client.Database("myDB")
+   coll := database.Collection("myCollection", opts)
+
+Read Preference
+---------------
+
+The read preference option describes how MongoDB clients route read
+operations to the members of a replica set. By default, an application
+directs its read operations to the primary member in a replica set.
+
+Read preference consists of the read preference mode and, optionally, a
+:rapid:`tag set list </core/read-preference-tags/>`, the
+:rapid:`maxStalenessSeconds </core/read-preference-staleness/>` option, and the
+:rapid:`hedged read </core/read-preference-hedge-option/>` option.
+
+Options
+~~~~~~~
+
+The {+driver-long+} provides the ``readpref`` package to handle
+read preference specification. You can set a read preference using the
+``SetReadPreference()`` method with an ``Option`` type. The ``Option``
+type has the following methods to specify the read concern:
+
+.. list-table::
+   :widths: 25 75
+   :header-rows: 1
+
+   * - Method
+     - Description
+
+   * - ``Nearest()``
+     - Specifies reads from a random eligible replica set member, irrespective of
```

Comment:
Optional: In all of these descriptions, "Specifies reads" is a little clunky. Could it say "All operations read..." or "Operations read..." or "The client reads..." or something like that?

Reply:
I changed the Write Concern descriptions as well to match now that all the other sections use this format.